bchrpc/log.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2013-2016 The btcsuite developers
```

Comment:
This is a new file, is the copyright still for them?

Reply:
Well code is mostly the same